src/Umbraco.Cms.Integrations.DAM.Aprimo/Models/ViewModels/AprimoAssetViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+namespace Umbraco.Cms.Integrations.DAM.Aprimo.Models.ViewModels
+{
+    public class AprimoAssetViewModel
+    {
+        public string Id { get; set; } = string.Empty;
+
+        public string Title { get; set; } = string.Empty;
+
+        public string Thumbnail { get; set; } = string.Empty;
```

Comment:
How would you get different sized images for rendering on the front-end?  Currently I can only see rendering the thumbnail, but it could be we'd want the full-size version or different crops.  Is any of that available?